__tests__/buildTranslationFiles.spec.ts

```diff
@@ -259,20 +262,12 @@ describe.each(formats)('buildTranslationFiles in %s', (fileFormat) => {
               end: 2,
               prefix: 'site-header.navigation.route.nested',
             }),
-          },
-          todos: {
-            ...generateKeys({ end: 2, prefix: 'numbers' }),
+            ...generateKeys({ end: 2, prefix: 'todosPage.numbers' }),
```

Comment:
What's changed here?

Reply:
Like I said in the PR description:
> 'should work with read' no longer produces the same result, but I think it was an error (it's not scoped).

```html
<ng-container *transloco="let t; read: 'todosPage.numbers'">
  <h1>ddsds</h1>
  {{ t('1') }}
```

I think there was an error here, so we'll make another file, when there's no scope declared?
`read` is just there to prefix all the translations below, not a scope.
I think we're behaving strangely here.

Reply:
Hm.. I see what you mean.
I'll take another look at the test. tnx

Reply:
This is definitely a regression, the test is defined correctly.
The thing here is that there is a [scope](https://github.com/jsverse/transloco-keys-manager/blob/infra/migrate-to-esm/__tests__/read/nested-scope.ts) called `todos-page` with no alias configured so by default the scope name is resolved as `todosPage`.
That means there is a regression here somewhere as the test made sense, the first part of the read represents the scope and should map the keys to it.
we need to revert your change and debug the issue.

Reply:
Okay, I'll try to look at it today.

Reply:
I have found a solution.

The declaration file, which I hadn't seen, isn't correct.
`TRANSCOLO_SCOPE` of type `TranslocoScope = ProviderScope | string | undefined` does not accept arrays.

If it worked before, I think it was a mistake, and `transloco` must have changed it.